setup.py

```diff
@@ -35,5 +35,9 @@
         'Programming Language :: Python :: Implementation :: CPython',
         'Programming Language :: Python :: Implementation :: PyPy',
     ],
-    py_modules=["jdcal"]
+    # py_modules=["jdcal"],
+    packages=find_packages("src"),
```

Comment:
As the package has no subpackages, `find_packages` is overkill, IMO:

```suggestion
    packages=["jdcal"],
```

Reply:
Won't some of these suggestions change since we have decided to go with the latest recommendation 
of using `setup.cfg` to specify such information and keeping `setup.py` to the minimum?

Reply:
When adjusting `setup.cfg` there will be a choice of using `find_packages` or listing the packages. IMO, listing them is better as the layout is relatively simple. So I think the comment still applies, but just to the new declarative syntax.